client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -479,15 +485,15 @@ export class SeoForm extends React.Component {
 }
 
 const mapStateToProps = ( state ) => {
-	const selectedSite = getSelectedSite( state );
+	const site = getSelectedSite( state );
```

Comment:
Is this renamed to make it more concise?

Reply:
Mostly because it was annoying me that it wasn't consistent with the other props. 😅 Should I revert it to keep the changes down?

Reply:
I'm not sure. Consistency is a valid reason to rename it too! It gets renamed to `selectedSite` again later on. Would we want to change that too?